### DISCUSSION TITLE
If it is possible to fetch some particular field all the time

### DISCUSSION THREAD

**Discussion by jeyashree-n** [sequelize/sequelize#14074] [Help & Questions] [answered]
I have a doubt If it is possible to fetch some particular field all the time or in all the fetch without mentioning the attribute name in all the fetch.

If there is any way to fetch automatically that field all the time?

Example: 
Table Name: member
Field: id, name, email, address, mobile number,..., etc.

In this, if I put member.findAll({ attributes: ['id','name'] }) alone then the "email" field all should be fetched.

**Comment by ephys** [comment, marked as answer]
I don't think scopes would work here because setting the `attributes` option field would override the one from the scope but you should be able to achieve what you're trying to do using the [`beforeFind` hook](https://sequelize.org/v6/manual/hooks.html): It'll receive the options and you can add the attributes you want to them.

I think it would look like this, but this code hasn't been tested:

```typescript
YourModel.beforeFind((options) => {
  if (Array.isArray(options.attributes) && !options.attributes.includes('email')) {
    options.attributes = [...options.attributes, 'email'];
  }
});
```
